const.go

```diff
@@ -6,11 +6,11 @@ const (
 	ReplaceRequest   = 3
 	UpdateRequest    = 4
 	DeleteRequest    = 5
-	CallRequest      = 6 /* call in 1.6 format */
+	Call16Request    = 6 /* call in 1.6 format */
 	AuthRequest      = 7
 	EvalRequest      = 8
 	UpsertRequest    = 9
-	Call17Request    = 10
+	CallRequest      = 10 /* call in 1.7 format */
```

Comment:
I don't think there are many 1.7 users and it could be not obvious for someone that 1.8, 1.9, 1.10 and the whole 2.x family included here. It we want to write a comment, we should add something like "call in >1.7 format"

Reply:
Done.